### PR TITLE
update documentation to add new can_upgrade permission

### PR DESCRIPTION
**Pull request by njogz**
Per #medic/cht-core/issues/7139

### CODE REVIEW

File: content/en/apps/reference/app-settings/user-permissions.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Each permission is defined as an array of user role identifiers that have the pe
 | `can_aggregate_targets` | Allows access to Target Aggregates page |
 | `can_bulk_delete_reports` | Allows users to select multiple reports and delete |
 | `can_configure` | Allows update of configuration parameters |
+| `can_upgrade` | Allows upgrade of the CHT instance |
```
Comment (garethbowen) [inline content/en/apps/reference/app-settings/user-permissions.md:26]: `upgrade` here could be mistaken for workflow upgrades, ie: configuration. How about we change it to...

```suggestion
| `can_upgrade` | Allows upgrades of the CHT Core Framework version via the API or admin interface |
```

**Review by garethbowen — CHANGES_REQUESTED** [review]
One minor change suggested.

**Review by garethbowen — APPROVED** [review]
Thanks!

**Comment by mrjones-plip** [comment]
@njogz  - can you merge to `main` if this is ready to go? Thanks!

**Comment by mrjones-plip** [comment]
Oh - oops! Maybe you're waiting for code issue to close? I thought it had passed AT, but I see now it's back in dev for some improvements.   No biggie!

**Pull request merged**